Extensions/shuffle+.js

```diff
@@ -463,15 +463,34 @@
 
 		await Spicetify.Platform.PlayerAPI.clearQueue();
 
-		Spicetify.Platform.PlayerAPI.addToQueue(
-			list.map(uri => ({
-				uri,
-				provider: "context",
-				metadata: {
-					is_queued: "false"
-				}
-			}))
-		);
+		{
+			const { _queue, _client } = Spicetify.Platform.PlayerAPI._queue;
+			const { prevTracks, queueRevision } = _queue;
+
+			const nextTracks = list.map(uri => ({
+				contextTrack: {
+					uri,
+					uid: "",
+					metadata: {
```

Comment:
if there's no uid, it doesn't need to be set

Reply:
Actually, it has to be there, The internal Spotify methods that work on this data don't null check, instead, when they call a function they use a fromPartial that fills the rest of the object with default values. Without the default values, you are guaranteed to run into errors.